docs/topics/patching-nodes.md

```diff
@@ -0,0 +1,40 @@
+# Patching Cluster Nodes
+
+To favor a consistent behavior, AKS Engine's approach to infrastructure management is immutability.
+By default, cluster nodes are not patched on bootstrap nor on a regular basis. 
+
+To prevent node patching, AKS Engine creates file `/etc/apt/apt.conf.d/99periodic` with the following content:
```

Comment:
Would users have to change this file then on top of setting linuxProfile.enableUnattendedUpgrades or linuxProfile.runUnattendedUpgradesOnBootstrap to true? 

How does this file affect using Kured?

Reply:
Sorry to clarify, so to use Kured, users must also set `linuxProfile.enableUnattendedUpgrades` to `"true"`?

Reply:
It depends what "to use Kured" means. It can be deployed regardless, but Kured will only orchestrate node reboots if reboots are required after a node patch.

Reply:
Got it, so they're used in conjunction. First enable node patching by setting `linuxProfile.enableUnattendedUpgrades` to `"true" (or `linuxProfile.runUnattendedUpgradesOnBootstrap` to `"true" if they prefer).  

Then node patches will likely require reboots, that's where Kured comes in.

Reply:
right